learn/on-premise/gcp/install-prerequisites.md

```diff
@@ -69,10 +90,37 @@ sudo apt-get install python3 -y
 
 ## RHEL
 
-Install below softwares on Platform Instance and StudioWorkspace Instance / AppDeployment Instance
+### The ssh user has privileges(root/sudo) or user don't have privileges for install/upgrade utility softwares
+
+#### Platform Instance
+
+- If given ssh user has privileges(root/sudo) or user don't have privileges to install/upgrade. WME Installer will automatically install the Docker software. Internet is not required for Installation in this case.
```

Comment:
why is there so much duplication of content in these pages?

this sentence occurs in many pages and is wrong in all the places.